app/views/shared/_show_products.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+<h2> <center><%= label %></center></h2>
+  <div class="row">
+  <% sort_by.each do |product| %>
+    <div class="col-md-4">
+      <div class="thumbnail">
+        <a href="<%= product_path(product.id)%>">
+          <%= image_tag(product.photo_url, class: "img-rounded product-thumb") %>
+          <div class="caption">
```

Comment:
We do have all closed divs, the ``</div>`` for ``<div> on line 9 is on line 22. For 6 opening div we have 6 closing ones